test/request_cache_plug_test.exs

```diff
@@ -56,7 +56,7 @@ defmodule RequestCachePlugTest do
 
     match "/my_route" do
       conn
-        |> RequestCache.store(:timer.seconds(20))
+        |> RequestCache.store()
```

Comment:
Could we add a second cache case here instead to test both?

Reply:
Ok I duplicated the test.